src/models/artboard/README.md

```diff
@@ -0,0 +1,66 @@
+# ArtBoard Model
+
+ArtBoard Model is responsible for storing the structure of the class and its parameter used for artBoardDraw funtionality.These classes are called by the `ArtBoardSketch.tsx` and `ArtBoardTurtle.tsx` There are majorly 4 different classes.
+
+<ul>
+<li> ArtBoardDraw.ts
+<li> ArBoard.ts
+<li> ArBoardManager.ts
+<li> Turtle.ts
+</ul>
+
+---
+
+&nbsp;
+
+## ArBoardDraw.ts
+
+Responsible for storing the thickness and the color of the line drawn on the artBoard. Each instance of `artBoardSketch.tsx` calls artBoardDraw constructor with color (RGB format) as the parameter.
+
+| Parameter    | Type                   | Description                                              |
+| ------------ | ---------------------- | -------------------------------------------------------- |
+| StrokeWeight | number                 | Defines the thickeness of the line drawn on the artBoard |
+| StrokeColor  | [number,number,number] | Defines the color of the line drawn on the artBoard      |
+
+Both of these parameters have their own getter and setter to maintain encapsulation.
+
+```
+constructor(strokeColor : [number,number,number]){
+
+}
+```
+
+## Turtle.ts
+
+Responsible for creating a turtle and each turtle is identified with their own unique id. Each instance of `artBoardTurtle.tsx` calls Turtle with the id, the initial position and the initial angle of the turtle.
```

Comment:
Responsible for creating a turtle; each turtle is identified with an unique id. Each instance of `artBoardTurtle.tsx` calls Turtle with an id, the initial x-y position, and the initial angle of rotation.